### PR TITLE
Restore ability to unit test views

### PR DESCRIPTION
**Pull request by bmagyarkuti**
Prior to release `3.0.1`, it was possible to unit test views. For example:
```js
test("test a view")
  .dataset("example_view")
  .input(
    "sample_data",
    "select 'hi' as col1, 1 as col2, 3.5 as col3, true as col4, date '2020-07-23' as col5"
  )
  .expect(
    "select 'hi' as col1, 1 as col2, 3.5 as col3, true as col4, date '2020-07-23' as col5"
  );
```

Such a unit test always fails after `3.0.1` (including `3.0.2`). It produces the following error message:
`Dataset sample_data could not be found`. This happens regardless of whether the view has been defined.

This PR re-introduces the ability to unit test views.

I'd like to nominate @Ekrekr as reviewer.

### CODE REVIEW

**Comment by lveraszto** [comment]
+1

File: core/actions/test.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { StringifiedMap } from "df/common/strings/stringifier";
 import { ActionBuilder } from "df/core/actions";
 import * as table from "df/core/actions/table";
 import { ITableContext } from "df/core/actions/table";
+import * as view from "df/core/actions/view";
```
Comment (Ekrekr) [inline core/actions/test.ts:6]:
```suggestion
import {View} from "df/core/actions/view";
```

File: tests/integration/bigquery_project/definitions/test.js [hunk 2]
```diff
@@ -86,3 +86,13 @@ test("wrong row contents")
...
+
+test("test a view")
+  .dataset("example_view")
+  .input(
+    "sample_data",
+    "select 'hi' as col1, 1 as col2, 3.5 as col3, true as col4, date '2020-07-23' as col5"
+  )
+  .expect(
+    "select 'hi' as col1, 1 as col2, 3.5 as col3, true as col4, date '2020-07-23' as col5"
+  );
```
Comment (Ekrekr) [inline tests/integration/bigquery_project/definitions/test.js:98]: +newline at end of file

**Review by Ekrekr — CHANGES_REQUESTED** [review]
Thanks for the fix!

**Comment by bmagyarkuti** [comment]
@Ekrekr I've incorporated both your suggested changes. Is there anything else I need to do before this can be merged?

**Comment by Ekrekr** [comment]
Thanks - I've run the tests and they're all good, I just need you to sign the CLA before I can merge.

**Comment by barna-emarsys** [comment]
@Ekrekr, when I accepted your suggested changes, you became a co-author on that commit, and your @gmail address seems to be missing a CLA, so it seems you now need to sign the CLA. Sorry about this, maybe it was a bad idea to use Github's built-in "accept changes" functionality.

**Review by Ekrekr — APPROVED** [review]

**Comment by Ekrekr** [comment]
Huh weird, I see that too, but it also shows as me having already signed it when I follow the link. Will just merge through it!

**Pull request merged**